.github/CODEOWNERS

```diff
@@ -35,47 +35,46 @@
 # (Reference: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners)
```

Comment:
Can you put a general catch-all at the very top to assign everything to @oppia/core-reviewers (with explanatory comment)? That way it will catch anything that doesn't get subsequently assigned.

Though do we have some kind of check already that ensures that every file in the codebase is covered here? If we do then we don't need that.

Reply:
Yes, we have a lint check to make sure that every file is in CODEOWNERS. See https://github.com/oppia/oppia/blob/7cda99aa968a4adf4cfdcb84e4de6e5456c39f4f/scripts/linters/codeowner_linter.py#L188